### DISCUSSION TITLE
Plotting Apollo MTR-1 radar XY coordinate data

### DISCUSSION THREAD

**Discussion by DaveTSG** [dbuezas/lovelace-plotly-graph-card#439] [Q&A] [answered]
Hello,

I've bought an [Apollo MTR-1 radar](https://apolloautomation.com/products/mtr-1), which outputs the X and Y coordinates of a target.

I would like to have two separate displays:

1. A simple dot on a standard cartesian plot for the instantaneous X/Y coordinate data. e.g. X-coordinate being the X axis, Y-coordinate on the Y-axis. As it would be instantaneous, it should only show the current data, not historical data. It would need to update itself rapidly so it reflects current data. Edit: [here's the solution to this problem](https://wiki.apolloautomation.com/books/mtr-1-ImK/page/zone-configuration-using-home-assistant).
2. Some other way of visualising this data over time. I have no idea what this could/should look like, I'm just wondering what ideas anyone here might have.

Any ideas please on how to do both of these?

Thank you.

**Comment by DaveTSG** [comment, marked as answer]
> 1. A simple dot on a standard cartesian plot for the instantaneous X/Y coordinate data. e.g. X-coordinate being the X axis, Y-coordinate on the Y-axis. As it would be instantaneous, it should only show the current data, not historical data. It would need to update itself rapidly so it reflects current data.

Apologies, it turns out that Apollo already has an example of this, found [here](https://wiki.apolloautomation.com/books/mtr-1-ImK/page/zone-configuration-using-home-assistant).

**Reply by dbuezas** [reply]
Oh wow, they use my card in their examples, I'm flattered

**Comment by DaveTSG** [comment]
I guess for #2, I probably need to just have something take screenshots of the plot for #1 and then just turn it into a timelapse.

I'm guessing this won't be easy to do so I'm going to forget the idea and mark this issue as answered.

**Reply by dbuezas** [reply]
You could try a scatter3d or a histogram2d maybe.

**Reply by DaveTSG** [reply]
> You could try a scatter3d or a histogram2d maybe.

Thanks, but I'm finding this add-on very difficult to understand.
For instance, I searched for scatter3d on the Plotly website as [you told me to do back in January](https://community.home-assistant.io/t/plotly-interactive-graph-card/347746/550?u=davetsg) and found [this](https://plotly.com/javascript/3d-scatter-plots/), which gives me a JS code example. How do I actually get that into Home Assistant, which uses YAML instead?

All the examples shown in [the add-on documentation](https://github.com/dbuezas/lovelace-plotly-graph-card) are all custom card/YAML format. So I'm not sure how I take the Javascript code and convert it to YAML?

**Reply by dbuezas** [reply]
In the readme of this repo there's a link to "the index of examples with images". Click it and look there

**Reply by DaveTSG** [reply]
Ahhhhhh I see now. I hadn't realised that was a link before - I thought it was just a heading, so I completely overlooked that.

Thank you for your help.

**Reply by dbuezas** [reply]
Haha, how ironic, I made the header a link for visibility
